.github/workflows/cve_checks.yml

```diff
@@ -1,5 +1,8 @@
 name: "Infra: CVE checks"
 on:
+  pull_request_target:
```

Comment:
I think we don't need pull_request_target for this particular workflow, `pull_request` should suffice.

Reply:
@yeikel 👀

Reply:
No, I think that we should use `pull_request_target`. If we add `pull_request` a malicious actor can still modify this workflow and potentially leak secrets from the project. In fact, we should probably have `pull_request_target` for all the workflows

Reply:
It's the other way around - the target one grants an access to secrets

Reply:
Looking a bit more into it, it seems that you are right and that `pull_request` is generally safer

Reply:
Namely this part 

> only use pull_request_target for workflows where access to secrets or write permissions is strictly necessary, such as commenting, labeling, or status updates on pull requests.

Reply:
I updated it with https://github.com/kafbat/kafka-ui/pull/745/commits/1300d0c52800d3ff1936fee05bf21759181be1ea

Thanks!